src/components/elements/AddUser/AddUser.tsx

```diff
@@ -10,119 +20,336 @@ interface FuncProps {
-                        </Form.Control.Feedback>
-                    </Form.Group>
-                </Row>
-
-                <Row className="mb-3">
-                    <Form.Group as={Col} sm={12} lg={4} controlId="formGridFirstname">
-                        <Form.Label>First name</Form.Label>
-                        <Form.Control required type="text" placeholder="Enter First name" />
+                            Please choose a Department.
+                        </Form.Control.Feedback> */}
+                                <Autocomplete
+                                    freeSolo
+                                    multiple
+                                    id="institutions"
+                                    disableClearable
+                                    value={selectedDepartments}
+                                    options={adminDepartments.map((option) => option.departmentLabel)}
+                                    onChange={(event, value) => SetSelectedDepartments(value as string[])}
+                                    renderInput={(params) => (
+                                        <CssTextField
+                                            variant="outlined"
+                                            {...params}
+                                            InputProps={{
+                                                ...params.InputProps,
+                                                type: 'search',
+                                            }}
+                                        />
+                                    )}
+                                />
+                            </Form.Group>
+                        </Row>
+                        {/*  <Row className="mb-3">
+                    <Form.Group as={Col} sm={12} lg={4} controlId="formGridDivision">
+                        <Form.Label>Division</Form.Label>
+                        <Form.Control required type="text" placeholder="Enter Division" value={division} name="division" onChange={(e) => handleValueChangeTargetValue("division", e)} />
                         <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
                         <Form.Control.Feedback type="invalid">
-                            Please choose a First name.
+                            Please choose a Division.
                         </Form.Control.Feedback>
+
                     </Form.Group>
+                </Row> */}
 
-                    <Form.Group as={Col} sm={12} lg={4} controlId="formGridMiddleName">
-                        <Form.Label>Middle name</Form.Label>
-                        <Form.Control type="text" placeholder="Enter Middle name" />
+                        <Row className="mb-3">
+                            <Form.Group as={Col} sm={12} lg={12} controlId="formGridRole">
+                                <Form.Label>Role(s)*</Form.Label>
+                                {/* <Select
```

Comment:
let's remove the comments if this code isn't used anymore

Reply:
Ok. Just left it as it is UI design is not finalized.  Paul wanted to build this. But he is open to new ideas at the same time if any.

Reply:
I will clean up all the comment code.

Reply:
Fixed. Please check new commit

Reply:
Fixed. Please check new commit.